### PR TITLE
Add vault.cryptomator to make vaults readable by Cryptomator apps.

### PR DESCRIPTION
**Pull request by ylangisc**

### CODE REVIEW

File: cryptomator/src/main/java/ch/cyberduck/core/cryptomator/CryptoVault.java [hunk 1]
```diff
@@ -153,6 +160,15 @@ public synchronized Path create(final Session<?> session, final String region, f
         if(log.isDebugEnabled()) {
             log.debug(String.format("Write master key to %s", masterkey));
         }
+        // Create vaultconfig.cryptomator
+        final Algorithm algorithm = Algorithm.HMAC256(mk.getEncoded());
+        final String conf = JWT.create()
+            .withJWTId(UUID.randomUUID().toString())
```
Comment (dkocher) [inline cryptomator/src/main/java/ch/cyberduck/core/cryptomator/CryptoVault.java:166]: Use `UUIDRandomStringService`

File: cryptomator/src/main/java/ch/cyberduck/core/cryptomator/impl/CryptoFilenameV7Provider.java [hunk 2]
```diff
@@ -23,7 +23,7 @@
 
 public class CryptoFilenameV7Provider implements CryptoFilename {
 
-    private static final int NAME_SHORTENING_THRESHOLD = 220; // https://github.com/cryptomator/cryptofs/issues/60#issuecomment-523238303
+    public static final int NAME_SHORTENING_THRESHOLD = 220; // https://github.com/cryptomator/cryptofs/issues/60#issuecomment-523238303
```
Comment (dkocher) [inline cryptomator/src/main/java/ch/cyberduck/core/cryptomator/impl/CryptoFilenameV7Provider.java:26]: Unrelated change?

**Review by dkocher — CHANGES_REQUESTED** [review]

File: cryptomator/src/main/java/ch/cyberduck/core/cryptomator/impl/CryptoFilenameV7Provider.java (hunk 2 above)
Comment (ylangisc) [reply cryptomator/src/main/java/ch/cyberduck/core/cryptomator/impl/CryptoFilenameV7Provider.java:26]: No, is config parameter in `vault.cryptomator`.

**Review by dkocher — APPROVED** [review]
